extensions/amp-bodymovin-animation/0.1/amp-bodymovin-animation.js

```diff
@@ -95,6 +126,52 @@ export class AmpBodymovinAnimation extends AMP.BaseElement {
     }
     return true;
   }
+
+  play_() {
+    if (this.playingState_ == PLAYING_STATE.PLAYING) {
+      return;
+    }
+
+    const message = JSON.stringify(dict({
+      'action': 'play',
+    }));
+    this.iframe_.contentWindow./*OK*/postMessage(message, '*');
+    this.playingState_ = PLAYING_STATE.PLAYING;
+  }
+
+  pause_() {
+    if (this.playingState_ == PLAYING_STATE.PAUSED) {
+      return;
+    }
+
+    const message = JSON.stringify(dict({
+      'action': 'pause',
+    }));
+    this.iframe_.contentWindow./*OK*/postMessage(message, '*');
+    this.playingState_ = PLAYING_STATE.PAUSED;
+  }
+
+  stop_() {
+    if (this.playingState_ == PLAYING_STATE.STOPPED) {
+      return;
+    }
+
+    const message = JSON.stringify(dict({
+      'action': 'stop',
+    }));
+    this.iframe_.contentWindow./*OK*/postMessage(message, '*');
+    this.playingState_ = PLAYING_STATE.STOPPED;
+  }
+
+  seekTo_(timeVal) {
+    const message = JSON.stringify(dict({
+      'action': 'goToAndStop',
```

Comment:
`goToAndPlay` makes more sense since pause is called separately here. I actually suggest calling it `seekTo` but removing `pause` from here and doing the pause in the 3P code instead. (idea being knowledge of Lottie-specific names should stays in the 3P implementation and not the player-generic component)